engine/interpreter/src/test/scala/pl/touk/nussknacker/engine/definition/ProcessDefinitionExtractorSpec.scala

```diff
@@ -43,6 +43,12 @@ class ProcessDefinitionExtractorSpec extends FunSuite with Matchers {
       classDefinition.map(_.methods.keys) shouldBe Some(Set("someField", "toString"))
   }
 
+  test("extract type info from classes from allowed classes") {
+    val types = ProcessDefinitionExtractor.extractTypes(processDefinition)
+    val classDefinition = types.find(_.clazzName == Typed[AllowedClass])
```

Comment:
You have duplicated "from classes" in scenario name. Maybe it should be "extract type info from allowed classes"?

Reply:
Done